docs/080_faq/README.md

```diff
@@ -13,6 +13,26 @@ or
 npx --node-options="--disable-warning=DEP0040" pepr [command]
 ```
 
+## How does Pepr compare to Kyverno?
+
+Pepr has similarities but ultimately is very different than Kyverno.
+
```

Comment:
mission statements are not the same.

Pepr is about making operators as easy as possible.

Kyverno is about reporting, not building operators

Reply:
Added as first bullet point in differences section.